src/widget/widget.ts

```diff
@@ -316,6 +91,402 @@ export function init() {
     });
 }
 
+function createOutput({ URL_BASE }: SettingConstants) {
+  function searchItem(item: GeocoderFeature) {
+    const img = venueIcon(item);
+    const title = el('span', [item.name]);
+    const locality = el('span', [item.locality ?? ''], style.itemLocality);
+    const li = el('li', [img, title, locality], style.listItem);
+    li.role = 'option';
+    li.setAttribute('data-feature-id', item.id);
+    return li;
+  }
+
+  function venueIcon(item: GeocoderFeature) {
+    const icon = iconData(item.category);
+
+    // http://localhost:3000/assets/mono/light/transportation-entur/Bus.svg
```

Comment:
Is this comment here as an example?

Reply:
This is not the line you're looking for...